lib/opensusebasetest.pm

```diff
@@ -197,10 +197,12 @@ sub set_standard_prompt {
 sub select_bootmenu_more {
     my ($self, $tag, $more) = @_;
 
-    assert_screen "inst-bootmenu", 15;
+    # do not waste time waiting when we already matched
+    assert_screen 'inst-bootmenu', 15 unless match_has_tag 'inst-bootmenu';
+    stop_grub_timeout;
 
     # after installation-images 14.210 added a submenu
-    if ($more && check_screen "inst-submenu-more", 1) {
+    if ($more && check_screen 'inst-submenu-more', 0) {
```

Comment:
fail to understand from source code what `$mustmatch` parameter suppose to do , and it's description missed in function description. can you please elaborate what this magic 1 to 0 suppose to bring ?

Reply:
Simple timeout

Reply:
so `$mustmatch` is timeout measured in seconds ? and you setting it to 0 so it will not wait at all ? 
( not saying that it is wrong just asking )